### ISSUE TITLE
Display a functional transformation of payoff results at nodes

### ISSUE THREAD

**Issue by andersonad** [SilverDecisions/SilverDecisions#191]
Oftentimes it is useful when communicating to stakeholders under risk preference to show them the cash-equivalent of their alternatives based on their particular utility curves. Would it be possible to add an option that would change the display value of nodes based on a function? Example image below of what this might look like using the example from issue #190.

![display-transformation-example](https://user-images.githubusercontent.com/12083964/74383758-6d83e080-4db5-11ea-85c1-61e9b59d1071.png)

In this issue, a stakeholder with a log-utility has applied a transformation to the aggregate of their payoffs. However, for communicating with the stakeholder the value of the proposition, it is often more useful to be able to say, "20.5 currency" instead of "3.03". It would also be useful if these transformations at the node could be propagated through the sensitivity analysis, but that may be complicated to implement.

**Comment by andersonad** [comment]
Also as a note I'm looking to build experience working on web applications so if you can point me where in the code base I would look in order to implement this kind of change I can give it a shot and try to put together a pull request.

If it's just easier for you to handle it or you disagree with the feature then I'll leave it at that.

**Comment by bkamins** [comment]
@mwasiluk - do you think, given the changes you are working on, it is OK for @andersonad to try making the changes (I would assume that this will not be simple, but maybe @andersonad would want to try it just to get some practice).

**Comment by andersonad** [comment]
After returning to this comment a couple of years later and looking at it. I realized that this capability is self-evident using the already existing transform payoffs functionality. 

TLDR: apply transformPayoffs one more time with the inverse function.

These three trees show the following operations:
- Original table with original values
- Table after applying transformPayoffs log(p), which pushes down all rewards to the terminals and subsequently applies the transform
- Second table after applying transformPayoffs exp(p), which converts the aggregated payoffs back to dollars according to the specified log utility, thus accomplishing the "re-labeling" of the utility back to equivalent dollars for the individual.
![Screenshot from 2022-05-13 16-04-02](https://user-images.githubusercontent.com/12083964/168381677-741dd72e-8b3a-4a16-a5fc-547768917143.png)
